daisy/workflow/step.go

```diff
@@ -63,6 +64,10 @@ func (s *Step) stepImpl() (stepImpl, error) {
 		matchCount++
 		result = s.DeleteResources
 	}
+	if s.MergeWorkflow != nil {
```

Comment:
In this stepImpl() function, feel like using some reflection to count instead of our long "if ... ++, if ... ++, if ... ++" pattern here?